src/auth/Fingerprint.cpp

```diff
@@ -81,10 +81,14 @@ void CFingerprint::handleInput(const std::string& input) {
 }
 
 std::optional<std::string> CFingerprint::getLastFailText() {
-    return m_sDBUSState.message.empty() ? std::nullopt : std::optional(m_sDBUSState.message);
+    if (!m_sFailureReason.empty() && m_sPrompt.empty())
```

Comment:
no need to check for prompt i think. same below

Reply:
yeah you're right, originally I wasn't clearing one when setting the other, but I think now it should work the same without this check